pkg/bff/gds.go

```diff
@@ -530,9 +534,13 @@ func (s *Server) GetCertificates(ctx context.Context, testnetID, mainnetID strin
 			return nil, nil
 		}
 
+		if _, ok := ctx.Deadline(); !ok {
+			ctx, _ = utils.WithContext(ctx)
+		}
```

Comment:
Interesting, can we not just pass in the context that's given to the func? Basically leaving it up to the caller to decide what's in the context.

Reply:
Fixed